src/Smuggler/Plugin.hs

```diff
@@ -89,10 +89,9 @@ unusedLocs (L (RealSrcSpan loc) decl, used, unused)
     = []
 
     -- Nothing used; drop entire decl
-    -- TODO: optimize
     | null used = [ (lineNum, colNum)
                   | lineNum <- [srcSpanStartLine loc .. srcSpanEndLine loc]
-                  , colNum <- [1..100]
+                  , colNum <-  [srcSpanStartCol loc .. srcSpanEndCol loc]
```

Comment:
I believe `srcSpanEndCol` here should give us the end column for the `SrcSpan`. Somehow it always yields `79` so I wonder if it somehow defaults to that. I asked about it on irc so let's see if someone has the answer. If I hardcode this value to say 100 everything works so I am not sure why this function does not give us the correct column end.

Reply:
Oh I found what was the problem. It picks up the last import line column. I'll need to read all line ends and then use the maximum.